internal/ir/xds.go

```diff
@@ -1108,3 +1112,40 @@ type SlowStart struct {
 	// Window defines the duration of the warm up period for newly added host.
 	Window *metav1.Duration `json:"window" yaml:"window"`
 }
+
+// TLSBundle contains tls certificate, private key and ca file in []byte format.
```

Comment:
can we reuse https://github.com/envoyproxy/gateway/blob/d0c12ebed88bf4f6244230e96ebc9c1670517b1a/internal/ir/xds.go#L277 ? and introduce a new `CACertificate` field in there ?

Reply:
yes, will update